### DISCUSSION TITLE
Need help figuring out how to make forms work

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#7957] [Help]
I have a database with 2 tables in it, 1 table is called locations, and has 2 fields in it -- Location ID and Location, thusly

![image](https://user-images.githubusercontent.com/24684691/192155905-b75cb94e-8f04-4b6d-94f6-ac4e036e45fc.png)

the values in this table will be used to populate a combo box in another screen,

Im not sure how to build the form to populate it, The Location ID field should fill automatically (even invisibly if possible) and I should be able to type in the location field -- 

![image](https://user-images.githubusercontent.com/24684691/192155977-ebbc4284-d146-4f23-8737-2e701c96657f.png)

This is what I get when I try but I suspect I have a deeper problem -- can someone point me to a solution?

Thanks
TIM

**Comment by melohagan** [comment]
Hey Tim, 

I replied to you on Discord: https://discord.com/channels/733030666647765003/1023313656676626472

Basically I think you're just missing the location ID. This docs link might help: https://docs.budibase.com/docs/actions#save-row

In future would you mind keeping the discussion to either discussions or the community channel. Responses aren't necessarily going to be immediate or any faster either way. 
With that said feel free to reply in either channel!

**Comment by taholmes160** [comment]
be glad to -- is that for forum or discord?  I thought I was in
community-help on discord

**Reply by melohagan** [reply]
It is community help on Discord, but I may answer some questions here and there!
